FirebaseAdmin/FirebaseAdmin.IntegrationTests/FirebaseAuthTest.cs

```diff
@@ -408,4 +415,18 @@ internal static RandomUser Create()
             };
         }
     }
+
+    /**
+     * Additional Xunit style asserts that allow specifying an error message upon failure.
+     */
+    internal class AssertWithMessage
```

Comment:
May be make this a private static class to further reduce the visibility? I don't feel too strongly either way. So your call.